### PR TITLE
Hox fix 1

### PR DESCRIPTION
**Pull request by Globalika**
fixed selected cell border appearance
fixed label to fit
rearrange files at appropriate positions

### CODE REVIEW

File: CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift [hunk 1]
```diff
@@ -101,6 +101,19 @@ extension CountriesController {
...
+                            willDisplay cell: UITableViewCell,
+                            forRowAt indexPath: IndexPath) {
+        guard let cell = cell as? CountryViewCell else { return }
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            if cell.isSelected {
+                cell.didSelect(indexPath: indexPath)
+            } else {
+                cell.didDeselect(indexPath: indexPath)
+            }
+        }
```
Comment (khrystyna-sb) [inline CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift:114]: And what will happen on iPhone?

Comment (khrystyna-sb) [reply CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift:114]: ok..

**Review by khrystyna-sb — APPROVED** [review]

**Review by khrystyna-sb — APPROVED** [review]

File: CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift [hunk 2]
```diff
@@ -101,6 +99,17 @@ extension CountriesController {
...
 
+    override func tableView(_ tableView: UITableView,
+                            willDisplay cell: UITableViewCell,
+                            forRowAt indexPath: IndexPath) {
+        guard let cell = cell as? CountryViewCell else { return }
+        if cell.isSelected {
+            cell.didSelect(indexPath: indexPath)
+        } else {
+            cell.didDeselect(indexPath: indexPath)
+        }
```
Comment (apolwork) [inline CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift:110]: Have you chance to try override `isSelected` in your cellClass?

File: CountriesOfTheWorld/CountriesOfTheWorld/Views/CountryViewCell.swift [hunk 3]
```diff
@@ -220,8 +224,8 @@ class CountryViewCell: UITableViewCell {
         static let decoratorShadowOffset = CGSize(width: 0, height: 4)
         static let decoratorShadowRadius: CGFloat = 4
         static let decoratorShadowOpacity: Float = 0.2
-        static let allowedNumberOfLinesToWrap = 3
-        static let labelMinimumScaleFactor = 0.5
+        static let allowedNumberOfLinesToWrap = 0
+        static let labelMinimumScaleFactor = 0.4
```
Comment (apolwork) [inline CountriesOfTheWorld/CountriesOfTheWorld/Views/CountryViewCell.swift:228]: This scale factor looks to small. Typically it is set to `0.8`

File: CountriesOfTheWorld/CountriesOfTheWorld/Views/DetailsFieldPlaceHolderView.swift [hunk 4]
```diff
@@ -104,8 +104,8 @@ class DetailsFieldPlaceHolderView: UIView {
     }
 
     private struct Constants {
-        static let labelAllowedLines = 3
-        static let labelMinimumScaleFactor: CGFloat = 0.5
+        static let labelAllowedLines = 0
+        static let labelMinimumScaleFactor: CGFloat = 0.4
```
Comment (apolwork) [inline CountriesOfTheWorld/CountriesOfTheWorld/Views/DetailsFieldPlaceHolderView.swift:108]: saa

File: CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift (hunk 2 above)
Comment (Globalika) [reply CountriesOfTheWorld/CountriesOfTheWorld/ViewControllers/CountriesController.swift]: yes i tried it and its work

**Review by ambrusha — APPROVED** [review]

**Pull request merged**